### PR TITLE
Song99BottlesLessonTest refactored

### PR DESCRIPTION
**Pull request by NataliyaUA**
Refactoring of Song99BottlesTest

### CODE REVIEW

**Review by SvetlanaG1011 — APPROVED** [review]
looks good to me

File: src/test/java/qa_java_beginners/NataliyaUASong99BottlesLessonTest.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        Assert.assertEquals(actualResult.toString(), expectedResult);
+
+      }
+
+
+    }
+
+
+
+
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/NataliyaUASong99BottlesLessonTest.java:90]: Unnecessary empty lines.

File: src/test/java/qa_java_beginners/NataliyaUASong99BottlesLessonTest.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        for(WebElement p : pAll) {
+            actualResult.append(p.getText());
+
+        }
+
+        Assert.assertEquals(actualResult.toString(), expectedResult);
+
+      }
+
+
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/NataliyaUASong99BottlesLessonTest.java:85]: Unnecessary empty lines.

File: src/test/java/qa_java_beginners/NataliyaUASong99BottlesLessonTest.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        List<WebElement> pAll = getDriver().findElements(pTags);
+
+        for(WebElement p : pAll) {
+            actualResult.append(p.getText());
+
+        }
+
+        Assert.assertEquals(actualResult.toString(), expectedResult);
+
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/NataliyaUASong99BottlesLessonTest.java:82]: You might get rid of this line.

**Review by anonymoustuna — APPROVED** [review]

**Pull request merged**